test/e2e/suite/conformance/certificates/suite.go

```diff
@@ -242,6 +242,31 @@ func (s *Suite) Define() {
 			Expect(err).NotTo(HaveOccurred())
 		})
 
+		It("should issue a certificate that defines a Common Name and Email Address", func() {
+			s.checkFeatures(CommonNameFeature)
+			s.checkFeatures(EmailSANsFeature)
+
+			testCertificate := &cmapi.Certificate{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "testcert",
+					Namespace: f.Namespace.Name,
+				},
+				Spec: cmapi.CertificateSpec{
+					SecretName: "testcert-tls",
+					CommonName: "test-common-name",
+					EmailSANs:  []string{"alice@example.com"},
+					IssuerRef:  issuerRef,
+				},
+			}
+			By("Creating a Certificate")
+			err := f.CRClient.Create(ctx, testCertificate)
+			Expect(err).NotTo(HaveOccurred())
+
+			By("Waiting for the Certificate to be issued...")
+			err = f.Helper().WaitCertificateIssuedValid(f.Namespace.Name, "testcert", time.Minute*5)
```

Comment:
This method will need extending to actually check for the presence/validity of the specified `emailSANs`. I think once you do this, you'll find the test case is failing for some issuer types (definitely ACME, possibly Vault).